packages/backend/src/indexers/CurrentDiscoveryIndexer.ts

```diff
@@ -0,0 +1,81 @@
+import { assert, Logger } from '@l2beat/backend-tools'
+import { ChainId } from '@l2beat/discovery'
+import { ChildIndexer } from '@l2beat/uif'
+import { UnixTime } from '@lz/libs'
+
+import { BlockNumberRepository } from '../peripherals/database/BlockNumberRepository'
+import { CurrentDiscoveryRepository } from '../peripherals/database/CurrentDiscoveryRepository'
+import { DiscoveryRepository } from '../peripherals/database/DiscoveryRepository'
+import { IndexerStateRepository } from '../peripherals/database/IndexerStateRepository'
+import { DiscoveryIndexer } from './DiscoveryIndexer'
+
+export class CurrentDiscoveryIndexer extends ChildIndexer {
+  readonly id = 'CurrentDiscoveryIndexer'
+
+  constructor(
+    private readonly blockNumberRepository: BlockNumberRepository,
+    private readonly discoveryRepository: DiscoveryRepository,
+    private readonly currentDiscoveryRepository: CurrentDiscoveryRepository,
+    private readonly indexerStateRepository: IndexerStateRepository,
+    private readonly chainId: ChainId,
+    logger: Logger,
+    discoverySnaphotIndexer: DiscoveryIndexer,
```

Comment:
`snaphot` 🔥 😄 

Do we need to sync/differentiate old `DiscoveryIndexer` by renaming this one also to something like `DiscoverySnapshotIndxer`?

Reply:
That was my idea, BUT the problem is that it has the `DiscoveryIndexer` ID in the DB. If we want to rename it, could be a separate PR